apps/test/util/testUtils.js

```diff
@@ -279,6 +279,30 @@ exports.createMouseEvent = function mouseEvent(type, clientX, clientY) {
 };
 
 /**
+ * Creates a key event of the given type with the additional parameters
+ * @param {string} type (keydown, keyup, keypress)
+ * @param {bool} keyCodeArg
+ * @param {bool} metaKeyArg
+ * @param {bool} altKeyArg
+ * @param {bool} shiftKeyArg
+ * @param {bool} ctrlKeyArg
+ */
+exports.createKeyEvent = function keyEvent(type, keyCodeArg, metaKeyArg, altKeyArg, shiftKeyArg, ctrlKeyArg) {
```

Comment:
Optional: reading about the deprecation of initEvent made me realize that passing in a `config` object with optional `metaKey`, `altKey`, etc. properties would probably be better here. Then you can just assign `altKey = config.altKey` below. Feel free to choose a different name besides "config"